### PR TITLE
Add mothball config and mothball assignments

### PR DESCRIPTION
**Pull request by eah13**

### CODE REVIEW

File: _includes/assignments.html [hunk 1]
```diff
@@ -1,7 +1,19 @@
+
+
 {% for ass in site.data.assignments %}
```
Comment (eah13) [inline _includes/assignments.html:3]: I giggled to myself multiple times when coding this block.

File: assignments/index.md [hunk 2]
```diff
@@ -5,11 +5,15 @@ weight: 7
...
 You will be asked to confirm acceptance of the assignment.
 Then you will be taken to a GitHub repo with instructions, source code, and other materials that will help you to complete the assignment. 
 Once you can view the repository, click the "Fork" button in the upper right corner of the page in order to begin the assignment. 
 
-{% include assignments.html %}
 
-{% include assignment-table.md %} 
\ No newline at end of file
+
+{% include assignment-table.md %}
```
Comment (eah13) [inline assignments/index.md:17]: I like the table at the top but that's just me.

**Comment by eah13** [comment]
I figured I better get this PR open before my changes were too hard to disentangle.  Hope it helps!  Anything else that should be mothball dependent can now trigger off of `site.mothballed`.

File: _includes/assignments.html (hunk 1 above)
Comment (jdmar3) [inline _includes/assignments.html:3]: Yeah. When I began working on that I did the same.

File: assignments/index.md (hunk 2 above)
Comment (jdmar3) [inline assignments/index.md:17]: It's better at the top. Makes more sense anyway.

**Pull request merged**